src/TritonRoute/src/gc/FlexGC_impl.h

```diff
@@ -217,14 +218,14 @@ class FlexGCWorker::Impl
 
   void checkMetalSpacing();
   frCoord checkMetalSpacing_getMaxSpcVal(frLayerNum layerNum,
-                                         bool isNDR = true);
+                                         bool checkNDRs = true);
   void myBloat(const gtl::rectangle_data<frCoord>& rect,
                frCoord val,
                box_t& box);
   void checkMetalSpacing_main(gcRect* rect,
-                              bool isNDR = true,
-                              bool querySpcRects = false);
-  void checkMetalSpacing_main(gcRect* rect1, gcRect* rect2, bool isNDR = true);
+                              bool checkNDRs = true,
+                              bool isSpcRect = false);
```

Comment:
It seems like these flags are going to be used pretty heavily it might be worth turning them into Flag enums. 

```c++
enum Value{
  EVEN                           = (1<<0),
  ODD                            = (1<<2),        
  ANOTHER_FLAG                   = (1<<3),        
  YET_ANOTHER_FLAG               = (1<<4),        
  SOMETHING_ELSE                 = (1<<5),        
  SOMETHING_COMPLETELY_DIFFERENT = (1<<6),
};
```

That way we can make this a single parameter with type checking. It would allow configuration with the following. `CHECK_NDRS | IS_SPECIAL_RECTANGLE`

Reply:
I'm not clear how this provides more type checking than the bool arguments do.  It could be more compact/readable.